politeiawww/user/cockroachdb/cockroachdb.go

```diff
@@ -741,6 +742,110 @@ func (c *cockroachdb) RegisterPlugin(p user.Plugin) error {
 	return nil
 }
 
+// EmailHistoriesGet see user.Database for details.
```

Comment:
These comments don't add any value.

Reply:
copy-pasted `interface` comments here, because there is nothing implementation-specific to add

although personally I don't like having this copy-paste (comments tend to get stale, copy-pasted comments ... that's ... ahaha :)